Tests/RegexTests/MatchTests.swift

```diff
@@ -574,9 +574,27 @@ extension RegexTests {
     // Inverted character class
     matchTest(#"[^a]"#,
               ("💿", true),
+              ("a\u{301}", true),
               ("A", true),
               ("a", false))
 
+    matchTest("[a]",
+      ("a\u{301}", false))
+
+    // CR-LF special case: \r\n is a single character with ascii value equal
+    // to \n, so make sure the ascii bitset optimization handles this correctly
+    matchTest("[\r\n]",
+      ("\r\n", true),
+      ("\n", false),
+      ("\r", false))
```

Comment:
@natecook1000 does this depend on matching semantics mode? That is, if we see `[\r\n]` is that either CR or LF, or is it exactly CR-LF, or does it depend on mode?